_includes/head.html

```diff
@@ -65,4 +65,5 @@
 
 <!-- Scripts
 ================================================== -->
-  <script src="{{ site.baseurl }}/assets/js/vendor/jquery-1.11.3.min.js"></script>
\ No newline at end of file
+  <script src="{{ site.baseurl }}/assets/js/vendor/jquery-1.11.3.min.js"></script>
```

Comment:
I just realized, why is this being included here in the head rather then with the rest of the scripts in the scripts file? The scripts should also be included at the bottom, not the head.